include/seahorn/Expr/Smt/Z3.hh

```diff
@@ -377,6 +377,13 @@ public:
 
   Expr eval(Expr e, bool completion = false) {
     assert(model);
```

Comment:
This commit needs to be split into multiple commits based on functionality added. For instance, updates that fix how structs are evaluated in a model should not be part of the semantics.

Reply:
This comment still remains. Git history needs to be reworked. I can help if needed.

Reply:
I will break it off before final review (after LLVM 10 merge).